SamplesDashboard/SamplesDashboard/Models/SampleQueries.cs

```diff
@@ -158,7 +171,14 @@ public class Repository
         [JsonProperty("dependencyGraphManifests")]
         public DependencyGraphManifests DependencyGraphManifests { get; set; }
 
+        [JsonProperty("defaultBranchRef")]
+        public Branch DefaultBranch { get; set; }
+
         public PackageStatus highestStatus { get; set; }
```

Comment:
naming style is inconsistent

Reply:
True, but that was pre-existing :D. Did you want me to fix all of the property names to Pascal case (like the majority of properties in that file), or should that be a separate code-cleanup PR?

Reply:
We can address this in a separate PR :)